### DISCUSSION TITLE
Woman Tech Day | Revive este gran evento

### DISCUSSION THREAD

**Discussion by FernandaOchoa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#312] [Announcements]
Te esperamos para revivir el Woman Tech Day y conmemorar este 8 de marzo

**[>>Clic aquí<<](https://youtu.be/LgPxevOtkEk)**

**Comment by doguedogue** [comment]
Felicidades 💃!

https://youtu.be/uaGM5SRStQo
